core/src/test/java/org/opensearch/sql/data/model/DateTimeValueTest.java

```diff
@@ -228,28 +186,9 @@ public void timestampWithVariableNanoPrecision() {
       assertEquals(LocalDate.parse(dateValue), timestampValue.dateValue());
       assertEquals(LocalTime.parse(timeWithNanos), timestampValue.timeValue());
       String localDateTime = String.format("%sT%s", dateValue, timeWithNanos);
-      assertEquals(LocalDateTime.parse(localDateTime), timestampValue.datetimeValue());
-    }
-  }
-
-  @Test
-  public void datetimeWithVariableNanoPrecision() {
```

Comment:
You can keep this test but with timestamp

Reply:
It exists above, ```  @Test
  public void timestampWithVariableNanoPrecision() {```
  Line 215